sql/updates/world/6.x/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE IF NOT EXISTS `scene_template` (
+  `SceneId` INT(10) UNSIGNED NOT NULL,
+  `Flags` INT(10) UNSIGNED NOT NULL DEFAULT '16',
+  `ScriptPackageID` INT(10) UNSIGNED NOT NULL,
+  `ScriptName` CHAR(64) NOT NULL DEFAULT '',
+  PRIMARY KEY (`SceneId`)
+) ENGINE=INNODB DEFAULT CHARSET=utf8;
```

Comment:
world tables are all set to myisam engine, please don't introduce inconsistencies here